common/src/main/java/com/wynntils/features/DiscordRichPresenceFeature.java

```diff
@@ -121,7 +123,11 @@ private void checkTerritory() {
     @Override
     protected void onConfigUpdate(ConfigHolder configHolder) {
         if (this.isEnabled()) {
-            Managers.Discord.load();
+            // This isReady() check is required for Linux to not crash on config change.
+            if (!Managers.Discord.isReady()) {
+                // Even though this is in the onConfigUpdate method, it is how the library is first loaded on launch
+                Managers.Discord.load();
+            }
```

Comment:
So basically it can't reload it? This method get's called 3 times on load.. Only loading once makes it not crash when changing settings?

Reply:
Seems like it, can you also test on your instance?